counsel-gtags.el

```diff
@@ -393,18 +398,20 @@ Use TYPE ∈ '(definition reference symbol) for defining global parameters.
 Use TAGNAME for global query.
 Use AUTO-SELECT-ONLY-CANDIDATE to skip `ivy-read' if have a single candidate.
 Extra command line parameters to global are forwarded through EXTRA-OPTIONS."
-  (let* ((root (counsel-gtags--default-directory))
-         (encoding buffer-file-coding-system)
-         (default-directory root)
-         (collection (counsel-gtags--collect-candidates
-                      type tagname encoding extra-options))
-         (ivy-auto-select-single-candidate t) ;; see issue #7
-         )
-    (if (and auto-select-only-candidate (= (length collection) 1))
-        (counsel-gtags--find-file (car collection))
-      (ivy-read "Pattern: " collection
-                :action #'counsel-gtags--find-file
-                :caller 'counsel-gtags--select-file))))
+  (if (> (length tagname) 0)
```

Comment:
You should use `string-empty-p` here instead, like `(unless (string-empty-p tagname) …)`.

The intention is more clear to whoever is reading the code

Reply:
Hi:
I made your correction. But prefered to invert the condition and use an if to show a message when an empty tagname.